src/app/shared/components/footer/footer.component.html

```diff
@@ -1,10 +1,41 @@
-<div class="footer">
-    <div>
+<section>
+    <div class="footer">
+        <div class="box">
+            <h3>Social Media</h3>
+            <div class="buttons-container">
+                <button mat-mini-fab color="white" aria-label="Example icon button with a menu icon">
+                    <a href="https://www.facebook.com/" target="_blank">
```

Comment:
The social media section should be centered, with the icons having the same height. The text should be in the same line as the icons.